src/utils/parse.js

```diff
@@ -20,6 +20,7 @@ function attachScope(node: Node) {
 
     case 'Function':
     case 'BoundFunction':
+    case 'GeneratorFunction':
```

Comment:
This makes me realize I have some cleanup to do. You had to add this in too many places 😞